tests/test_scanner.py

```diff
@@ -35,3 +37,17 @@ def test_scanning_ner_human_language_bias():
     assert name == "ner-human-language-bias"
     assert kind == "bias"
     assert len(df) > 50
+
+
+@pytest.mark.skipif(
```

Comment:
I'm skeptical of a unit test that can't run in a CI/CD pipeline. If we skip this test in automated runs then we will effectively be blind to any regressions that are introduced in between manual runs. Is there a way to refactor the test so that it can run via actions?

Reply:
I have to think about this one. The big issue seems to be downloading the chat model. Even the small one is quite large (~20 GB).

Reply:
Could we use something like `Mock` to provide outputs similar to what the model would provide in order to be able to test the scanner without having to fully download the chat model?